automated/health-checks/tests/health-check-test-002.yaml

```diff
@@ -0,0 +1,23 @@
+metadata:
+    name: Test Health Check 002
+    format: "Lava-Test Test Definition 1.0"
+    description: "Standard Health Check for RevPi devices"
+    maintainer:
+        - r.gsponer@kunbus.com
+    os:
+        - raspian
+    scope:
+        - functional
+    devices:
+        - Core
+        - Core 3
+        - core 3+
+        - Core S
```

Comment:
This should also work on Core SE, Compact and Flat. Even of my toaster if it had SSH and would run Linux :-)

Reply:
ah ok. thought this is just free-form text. If this is referencing to existing devices, then it makes sense